src/main/java/com/google/sps/servlets/UpdateServlet.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.ChromeOSDevice;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.sps.gson.Json;
+import com.google.sps.servlets.Util;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.Response;
+import org.json.simple.JSONArray;
```

Comment:
Imports should be in alphabetical order according to google's [java style guide](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing).